R/finalize_plot.R

```diff
@@ -519,23 +503,57 @@ finalize_plot <- function(plot = NULL,
 # Function to create plot object with left aligned legend on top
 buildChart <- function(plot,
                        consts,
-                       legend_build) {
+                       overrides,
+                       legend_build,
```

Comment:
Let's change to legend_shift as logical and remove legend elimination from here--maybe add to theme?